docs/features/kubernetes/configuration.md

```diff
@@ -441,6 +476,13 @@ rules:
       - get
       - list
       - watch
+  - apiGroups:
+      - metrics.k8s.io
+    resources:
+      - pods
+    verbs:
+      - get
+      - list
```

Comment:
Shouldn't we add `-watch` here as well?

Reply:
Actually `list` is the only verb Backstage ever uses. If you stare at this expression:

https://github.com/backstage/backstage/blob/b3fc6211936118dfeceeb3fe6ce767095b6342e5/plugins/kubernetes-backend/src/service/KubernetesFetcher.ts#L184-L190

It always ends in a plural

Reply:
@jamieklassen thanks for the clarification. I believe the `watch` is not really required, but I didn't touch the existing docs. I can make it consistent if it's more preferred.